CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ If anything is missing from the migration guide, please submit an issue.
 
 ### New Features
 
+Starting with version 7, the npm module has been changed from `cucumber` to `@cucumber/cucumber`
```

Comment:
I think was better where it was instead of under new features

Reply:
Let’s move it under CHANGED

Reply:
Oh, this was a bad merge. I've put it back to how it was.

We are already listing the package name change under the breaking changes heading. Do we want to rework the 7.0.0-rc.0 changes to use the new headings? We could pick this up in a separate pr/issue